src/v2/modules/invoice-accounts/routes.js

```diff
@@ -20,10 +22,28 @@ exports.getInvoiceAccounts = {
   path: '/crm/2.0/invoice-accounts',
   handler: controller.getInvoiceAccounts,
   options: {
-    description: 'Get a list of invoice accounts that have the speicified id values',
+    description: 'Get a list of invoice accounts that have the specified id values',
     validate: {
       query: {
-        id: Joi.array().single().items(Joi.string().uuid()).required()
+        id: Joi.array().single().items(validators.GUID).required()
+      }
+    }
+  }
+};
+
+exports.createInvoiceAccount = {
```

Comment:
For adding an entity you can now use the entityHandlers.createEntity function

An example for the address is https://github.com/DEFRA/water-abstraction-tactical-crm/blob/develop/src/v2/modules/addresses/routes.js#L10